src/main/resources/db/migration/V20231218_47__GA-47_new_access_types_table.sql

```diff
@@ -0,0 +1,14 @@
+-- create table
+-- PK/FK relationships in tables access_types
+-- access_types pk column = version
+-- automatic when creating bigserial:
+--   access_types_version_seq (Sequence)
+--   nextval('access_types_version_seq') NOT null
+
+CREATE TABLE access_types(
+	version bigserial not null,
+    access_type jsonb NOT NULL,
```

Comment:
```suggestion
    access_types jsonb NOT NULL,
```

should be `access_types` - jira is incorrect, should also be updated

Reply:
agree with Abul - need to clear up with Suman